### PR TITLE
Series view

### LINKED ISSUE #71: Serie Routes View
### REQUIREMENTS / DESCRIPTION

As a **user**, I want to view a series of routines, where each routine is represented as a separate route. Each routine route should display a title indicating the name of the routine, followed by a series of tables. Each table should contain the name of a series and its corresponding details.

#### Functionalities:

1. **Display Routine Name:**
   - Each routine route should prominently display the name of the routine at the top.

2. **Series Tables:**
   - For each routine, a series of tables should be displayed.
   - Each table should contain the name of a series and its corresponding details.

3. **Consolidated View:**
   - All series within a routine should be displayed on the same page.
   - Users can easily navigate between different routines by accessing their respective routes.

#### User Interaction:

- **Navigating Routines:**
  - Users can access different routines by navigating to their respective routes.
  - Each routine route will display the corresponding series of that routine.

- **Viewing Series Details:**
  - Users can view the details of each series within a routine by examining the corresponding tables.
  - Series details should include relevant information such as name, duration, repetitions, etc.

#### Design Considerations:

- The routine name should be prominently displayed to provide context to the user.
- Tables should be well-organized and visually appealing, making it easy for users to scan through series details.
- Consider using responsive design to ensure optimal viewing experience across devices.

#### Implementation Notes:

- Utilize HTML, CSS, and JavaScript to create the view for routine routes.
- Implement dynamic rendering to display series details based on the selected routine.
- Ensure consistent styling and layout across all routine routes for a cohesive user experience.

### Expected result
image

### PR DESCRIPTION
**Pull request by etormasa**
# Description

Implement views
Add js to make set green the div when checkbox is checked

## Screenshots

no spaces aligned
image

with spaces aligned 

image

### CODE REVIEW

**Review by PudinDeveloper — APPROVED** [review]
LGTM 🥥

**Review by angel23v — APPROVED** [review]

**Review by MauricioNC — APPROVED** [review]

**Review by BlazeDex — APPROVED** [review]

**Comment by BlazeDex** [comment]
Just fix conflicts.

File: app/views/exercises/_exercises.html.erb [hunk 1]
```diff
@@ -1,16 +1,26 @@
...
+  <div class="leftpart">
     <h3><%= exercise.repetitions %></h3>
-  </div>      
-  <div style="margin-right: 20px;">
     <h3><%= exercise.weight %></h3>
   </div>
-  <div style="margin-right: 20px;">
-    <%= link_to 'Edit', edit_routine_series_exercise_path(@routine, serie, exercise), class: 'btn__actions' %>
+  <div class="rightpart">
+    <div class="checkbox-wrapper-44 miCheckbox">
```
Comment (juperameza) [inline app/views/exercises/_exercises.html.erb]: Maybe change micheckbox to something more easy to understand when you see it 
What if there's another checkbox would it be yourcheckbox?
Remember to have good variable naming to have an easier time understanding your code

**Review by juperameza — CHANGES_REQUESTED** [review]

File: app/assets/stylesheets/pages/_routines.scss [hunk 2]
```diff
@@ -75,3 +75,267 @@
...
+
+.rightpart{
+  width: 60%;
+  justify-content: space-between;
+}
+
+.leftpart{
+  width: 40%;
+  justify-content: space-around;
+}
```
Comment (juperameza) [inline app/assets/stylesheets/pages/_routines.scss]: Same here
Left part right part

**Review by juperameza — CHANGES_REQUESTED** [review]

**Review by juperameza — APPROVED** [review]

**Pull request merged**